content/realtime/channel-metadata.textile

```diff
@@ -18,98 +17,27 @@ jump_to:
-
-h3(#occupancy-events). channel.occupancy lifecycle events
-
-When "channel.occupancy events are enabled":https://support.ably.com/support/solutions/articles/3000088326 for your app, you can subscribe to channel "occupancy":#occupancy events by attaching to the channel @[meta]channel.lifecycle@ using a realtime connection. However, please bear in mind that "realtime connections and channels have rate limits":https://support.ably.com/solution/articles/3000053845 based on your package size. If the rate of channel lifecycle events exceeds this, a realtime connection is not going to be a reliable way to consume these events as you may get disconnected or lifecycle events may be discarded.
-
-If you wish to stream this data beyond the limit of a single subscriber connection, you can make use of "Ably Reactor":/reactor with a "Reactor Rule":https://support.ably.com/solution/articles/3000074406. This allows you to react to lifecycle events and trigger "serverless functions":/general/events, "deliver messages to queues":/general/queues, or "stream data to third party services":/general/firehose for realtime processing.
-
-h3(#reactor). Reactor rules
-
-You can associate "Reactor rules":https://www.ably.io/reactor with metachannels in the same way as for regular channels; configure these using the "Message":/realtime/messages source and use a @channelFilter@ of, for example, @^\[meta\]log@. Normally the @channelFilter@ regex will only match normal, non-meta channels; however, a regex of the form @^\[qualifier\]pattern@ is special-cased to match channels with that qualifier. The @pattern@ is a regex as normal.
-
-If you want webhooks for channel lifecycle events, creating a "channel lifecycle webhook":/general/events#sources is an easy alternative to creating a message rule for the @[meta]channel.lifecycle@ metachannel; such webhooks will have simpler POST bodies as the lifecycle events won't be wrapped in "@Message@":/realtime/types#message objects.
+1. "Metachannels":/realtime/metachannels - Metachannels are special channels that provide metadata about all channels associated with an Ably application, this includes metadata associated with channel and connection lifecycle.
+2. "Inband channel occupancy events":/realtime/inband-occupancy - Inband occupancy events are occupancy events that are present on a user-created channel. 
+3. "Channel Status API":/rest/channel-status - The Channel Status API is a REST API that allows users to query a channel to retrieve channel metadata. It also allows you to "enumerate":/rest/channel-status#enumeration-rest all the active channels in a particular app. The details of the Channel Status API are covered in the "Channel Status documentation":/rest/channel-status. It is not recommended you poll for status updates as this is inefficient.
```

Comment:
"It is not recommended ... " - does this comment belong here, or in the section we're linking to?

Reply:
Having in both places may have been a bit over the top. I have removed that text.